core/src/main/java/io/questdb/std/Unsafe.java

```diff
@@ -50,6 +52,22 @@ public final class Unsafe {
     //#endif
     private static final AnonymousClassDefiner anonymousClassDefiner;
     private static final LongAdder[] COUNTERS = new LongAdder[MemoryTag.SIZE];
+    private static final AtomicLong OFF_HEAP_ALLOCATED = new AtomicLong(0);
+    private final static long HEAP_BREATHING_SPACE = 1L << 29; // 512 GiB
```

Comment:
these comments appear to be wrong. are they outdated? 
1 << 29 == 512 MB
1 << 26 == 64 MB

Reply:
yes, I'll update the comments

Reply:
done in https://github.com/questdb/questdb/pull/2279/commits/42d173b4c748280302ba33862d81a89080af7fd7